app/src/main/res/values/strings.xml

```diff
@@ -0,0 +1,4 @@
+<?xml version="1.0" encoding="utf-8"?>
+<resources>
+    <string name="expandable_bottomhalf">Expandable Bottomhalf</string>
```

Comment:
This should be in the main strings file, together with the other strings (ui/i18n folder)